### DISCUSSION TITLE
Función permiso web-admin-read

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#125] [2. Questions and Answers] [answered]
Se qué la función del web-admin-read te permite ver las colecciones de una compañía, pero me generó la siguiente duda:

Si una compañía, tiene kb instalado y usuarios como expertos y grupo de experto. Si le coloco el permiso web-admin-read, el usuario podrá ver administrador KB y administrador, ya que sin ese permiso no podrá ver el Administrador KB.

Entonces tengo la duda: ¿Por qué se debe visualizar ambos?, ya que existen los posibles casos:

1.  Un usuario puede ver el Administrador, pero no necesariamente sea un experto para tener el Administrador KB
2. Un usuario puede ver el Administrador KB por ser experto, pero no necesariamente debe ver la información de las colecciones (user, formularios, base de datos, etc)
3. Un usuario puede ver ambos, porque es experto y a la vez el administrador de la compañía
4.- Un usuario no puede ver ambos, porque es un usuario simple (rol default), que podría preguntar cosas a Knowledgebase.

Entonces creo que sería bueno que existiese el permiso: web-adminkb-read / web-adminkb-write para temas del Administrador KB y no depender del web-admin-read

Adjunto la imagen de ambos administradores

![image](https://user-images.githubusercontent.com/61419471/116588871-0edb5180-a8ea-11eb-86b0-83dbe88d6138.png)

Gracias y estaré atento

**Comment by acedward** [comment, marked as answer]
Suena, bien @NicolasPLG  el feature request. Sólo dos pequeños puntos:

1.  web-admin-read está deprecado, solo existe soporte para retro-compatiblidad. Por favor solo usar admin-* reports-* db-* etc. Aquí puedes encontrar los permisos: https://github.com/Cotalker/documentation/discussions/118 (la documentación oficial será actualizada pronto)
2.  KB es un módulo sin soporte de ingeniería para nuevas funcionalidades. Cualquier cambio por favor solicitar a tu representante de Cotalker @jdurandpaz para generar un ticket y realizar las mejoras y/o actualización

**Reply by NicolasPLG** [reply]
gracias @acedward por tu respuesta, auqnue me hacía útil el web-admin-read para ver el admin y no depende el admin-*-write o read

Sobre el KB Manager lo consultaré con @jdurandpaz
